adapter-base/src/main/resources/META-INF/native-image/org.eclipse.hono/hono-adapter-base/native-image.properties

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+
+Args = -H:+RunReachabilityHandlersConcurrently
```

Comment:
this needs to (only) be put into the concrete adapter implementation modules so that the native-build command applies it *last*. If put into one of the dependencies, it is uncertain whether it will be applied before *gax-rpc* sets it to the opposite value ...

Reply:
Done, I was a bit worried about the depency priorities for native-image build arguments but good to know that main application .jar takes first priority.